wordpress-mobile/WordPress-iOS
# Adds the "Jetpack powered" banner to the Notifications screen

Fixes #NA

This PR adds the "Jetpack powered" banner to the Stats and Notifications screens. This change only affects the WordPress app, not the Jetpack app

On iPad and iPhone regular landscape (e.g. iPhone 13 Pro Max), the FAB will adjust its position if the banner is visible

iPad Notifications Dark | iPhone Notifications Light
-- | --
<img width="450" src="https://user-images.githubusercontent.com/34376330/177015271-1b386bb3-9fb4-4e57-b461-0bf9727dfd43.png"> | <img width="400" src="https://user-images.githubusercontent.com/34376330/176941036-81ac840d-8b92-498e-a7fb-00706e918a91.png">


To test:
Test on iPhone, iPad and larger iPhone (e.g iPhone 13 Pro Max)
1.  Build/run this branch on the `WordPress` target
2. Tap on Notifications and make sure the banner is visible
3. Build the Jetpack target from this same branch, repeat steps 2 and 5 and make sure the banner does not appear in the Jetpack App

## Regression Notes
1. Potential unintended areas of impact
Low: this PR adds a new property to WPTabBarController, and make some runtime adjustments to view to account for the banner being visible/not visible

2. What I did to test those areas of impact (or what existing automated tests I relied on)
test views on different devices, quick check if there was something obviously wrong in the memory graph

3. What automated tests I added (or what prevented me from doing so)
None
PR submission checklist:

- [x] ~~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~~ RN are frozen for this release
